### PR TITLE
Fix/tao 10210/permission brs

### PR DESCRIPTION
**Pull request by jbout**
depends on : https://github.com/oat-sa/generis/pull/775/files
Use the generis filtering helper to correctly handle the scenario where no Data Access-control  is setup

### CODE REVIEW

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: manifest.php [hunk 1]
```diff
@@ -47,13 +47,13 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '37.3.1',
+    'version'     => '37.3.2',
```
Comment (boajer) [inline manifest.php:50]: Shouldn't it be minor version raised, as dependency raised minor version too ? 
```suggestion
    'version'     => '37.4.0',
```

Comment (mike-ionut-mihai-sandu-tao) [reply manifest.php:50]: this is a bug fix, in generis we added a feature

File: scripts/update/Updater.php [hunk 2]
```diff
@@ -2029,6 +2029,6 @@ public function update($initialVersion)
             $this->setVersion('37.3.0');
         }
 
-        $this->skip('37.3.0', '37.3.1');
+        $this->skip('37.3.0', '37.3.2');
```
Comment (mike-ionut-mihai-sandu-tao) [inline scripts/update/Updater.php:2032]:
```suggestion
        $this->skip('37.3.0', '37.4.0');
```

**Comment by boajer** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**